From 1c8e21e220ed0af6de3ab776d3edf8f0be69e779 Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Mon, 22 Dec 2014 09:45:08 +0100
Subject: [PATCH 1/2] Disable tests and manpages

Tests are forcibly statically linked to libopkg, which does not work in
a shared-only build.

Since Buildroot does not care about testsuites, just disable building the
tests. Disable manpages at the same time for the same reason.

Note: dynamically linking with libopkg is also possible, but that would
not be upstreamable (too much changes already in upstream).

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 8baa62c..497b774 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I shave -I m4
 
-SUBDIRS = libbb libopkg src tests utils man
+SUBDIRS = libbb libopkg src utils
 
 
 HOST_CPU=@host_cpu@
-- 
2.1.0

